tests/conftest.py

```diff
@@ -590,6 +586,11 @@ def _bridge_ports(node_interface):
     Get nodes active NICs.
     First NIC is management NIC
     """
+    if not is_baremetal_or_psi_cluster:
+        nodes_nics = {node_name: {"available": [], "occupied": nic} for node_name, nic in node_physical_nics.items()}
```

Comment:
@EdDev Please pay attention to this list assignment.
On one hand, this is harmless in any cluster >4.15, when OpenshiftSDN was totally abandoned. In OVNKubernetes using the primary interface is not allowed, because it serves as the br-ex port, so marking it as "occupied" is the right thing to do.
On the other hand - can we assume that cloud-providers clusters will always be >4.15 and/or with OVNKubernetes?
I believe we can, but your insight here will also be valuable. Thank you

Reply:
For context - this functionality is currently not needed on any version prior to 4.19